xml/decoders/uhlenbrock/PaneExtendedFmap.xml

```diff
@@ -115,9 +118,9 @@
 			<row>
 				<label>
 					<text>Mapping #1 - Conditions: </text>
-					<text xml:lang="de">Erweitertes Fkt.-Mapping #1 - Bedingungen: </text>
-					<text xml:lang="nl">Mapping #1 - Voorwaarden: </text>
-					<text xml:lang="cs">Mapování #1 - Podmínky: </text>
+					<text xml:lang="de">&lt;html&gt;&lt;strong&gt;Erweitertes Fkt.-Mapping #1 - Bedingungen: &lt;/strong&gt;&lt;/html&gt;</text>
```

Comment:
Why is this change only done to the translated texts, not the English text?

Reply:
I think the English text should also be updated with `&lt;html&gt;&lt;strong&gt;`